main.go

```diff
@@ -257,6 +259,10 @@ type Options struct {
 		DataOnly bool `cli:"--data-only"`
 	} `cli:"curl"`
 
+	Uuid struct {
+		Uuid string
```

Comment:
This is pedantic as hell and I'm sorry, but where the first letter of UUID is capitalized, golang linters will yell that it should be `UUID`, not `Uuid`. As such, I'd just prefer that we conform to that standard